src/main/java/seedu/address/logic/Logic.java

```diff
@@ -37,6 +38,9 @@
     /** Returns an unmodifiable view of the filtered list of items */
     ObservableList<Item> getFilteredItemList();
 
+    /** Returns an unmodifiable view of the sorted list of items */
+    SortedList<Item> getSortedItemList();
+
```

Comment:
Do we want a separate (from `getFilteredItemList()`) method here? Why not let sorting be done within `getFilteredItemList()`

Reply:
Good idea! Done that please check!

Reply:
Edit bugs out if we were to do that!

Reply:
Attempting to do it now

Reply:
Done! @halcon-blanco please check